Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py

```diff
@@ -167,45 +167,72 @@ def fetch_incidents():
     filter_alerts_kwargs = get_incident_init_params()
     incidents = []
     items = sixgill_alerts_client.get_actionable_alerts_bulk(limit=MAX_INCIDENTS, **filter_alerts_kwargs)
-    newest_incident_date = datetime.strptime(items[0].get('date'), DATETIME_FORMAT)
-    offset = 0
-    items_to_add = []
-    if newest_incident_date > last_fetch_time:
-        # finding all new alerts since last fetch time
-        while items:
-            for item in items:
-                if datetime.strptime(item.get('date'), DATETIME_FORMAT) > last_fetch_time:
-                    items_to_add.append(item)
-
-            if len(items_to_add) - offset == len(items):
-                offset += len(items)
-                items = sixgill_alerts_client.get_actionable_alerts_bulk(limit=MAX_INCIDENTS, offset=offset,
-                                                                         **filter_alerts_kwargs)
-            else:
-                items = []
-    demisto.info(f'Found {len(items_to_add)} new alerts since {last_fetch_time}')
-
-    # getting more info about oldest ~max_incidents_to_return(can be more because of sub alerts)
-    if len(items_to_add):
-        items_to_add.reverse()
-        newest_incident_date = items_to_add[-1].get('date')
-        for item in items_to_add:
-            item_info = sixgill_alerts_client.get_actionable_alert(actionable_alert_id=item.get('id'))
-            item_info['date'] = item.get('date')
-            new_incidents = item_to_incident(item_info, sixgill_alerts_client)
-            incidents.extend(new_incidents)
-            if len(incidents) >= max_incidents_to_return:
-                newest_incident_date = item.get('date')
-                break
-
-    demisto.info(f'Adding {len(incidents)} to demisto')
-    demisto.incidents(incidents)
-
-    if len(incidents):
-        demisto.info(f'Update last fetch time to: {newest_incident_date}')
-        demisto.setLastRun({
-            'last_fetch_time': newest_incident_date
-        })
+
+    if len(items) >= 0:
```

Comment:
```suggestion
    if len(items) > 0:
```
On the next line, you use `items[0]` which will cause "index out of range" if `len(items) == 0`